tests/test_repo/oca_dependencies.txt

```diff
@@ -0,0 +1,5 @@
+# List the OCA project dependencies, one per line
+# Add a repository url and branch if you need a forked version
+
+partner-contact https://github.com/OCA/partner-contact 8.0
```

Comment:
this is a test, not an example. The example is in the sample_files/oca_dependencies.txt